tests/unit_tests/data_steward/utils/pipeline_logging_test.py

```diff
@@ -0,0 +1,72 @@
+"""
+Unit test for pipeline_logging module.
+
+Ensures logging handlers are set up correctly by making sure:
+    1. The right number of expected handlers exists
+    2. The StreamHandler exists depending on test input
+    3. The log files that are configured point to the right file location
+
+Original Issue = DC-637
+"""
+
+import logging
+# Python imports
+import shutil
+import unittest
+from datetime import datetime
+
+import mock
+
+# Project imports
+from utils.pipeline_logging import generate_paths, create_logger
+
+
+class PipelineLoggingTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.log_file_list = ['path/', 'faked.log', 'path/fake.log']
+        self.log_path = [
+            datetime.now().strftime('path/curation%Y%m%d_%H%M%S.log'),
+            'logs/faked.log', 'path/fake.log'
+        ]
+        generate_paths(self.log_file_list)
```

Comment:
since we are using generate_paths() in the setup() we should be testing the function more granularly.

Reply:
`test_generate_paths` exists in the unit test already. Do I need to add anything else to it?